service/compute/api.go

```diff
@@ -386,50 +478,42 @@ func (a *ClustersAPI) DeleteByClusterIdAndWait(ctx context.Context, clusterId st
 // state will be rejected with an `INVALID_STATE` error code.
 //
 // Clusters created by the Databricks Jobs service cannot be edited.
-func (a *ClustersAPI) Edit(ctx context.Context, request EditCluster) error {
-	return a.impl.Edit(ctx, request)
+func (a *ClustersAPI) Edit(ctx context.Context, editCluster EditCluster) (*WaitGetClusterRunning[any], error) {
```

Comment:
This was not a long running op and now potentially is. That's odd.

Reply:
it was

Reply:
there was `EditAndWait` :) `Edit` does not trigger a wait anymore, because it just returns `WaitGetClusterRunning`, on which you have to call `.Get()`